teachopencadd/talktorials/019_One_Hot_Encoding/talktorial.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>This picture can't display in both of my computer and laptop. And for all fugures, I suggest resize them by <code>convert</code> from the terminal:</p><p><code>convert example.png -resize 200x100 example.png</code></p><p> Also suggest add all figures using the Markdown notation: </p><p><code>![Short figure title](images/filename.png)</code></p><p> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Could you please check it again ? </p><p>The image is present in the images folder under '_build/html' folder, so it should be seen in the html page.</p><p>Thank you.</p>

Reply:
<p>I know what happen :)</p><p>The name of the picture is 'logo.PNG', not 'logo.png'.</p>

Reply:
The picture still can't be shown in the notebook. Please correct the name of the picture file 'logo.PNG' to 'logo.png' in the folder. @sakshimisra